runtime/src/main/java/org/corfudb/recovery/RecoveryUtils.java

```diff
@@ -82,41 +81,33 @@ static ILogData getLogData(CorfuRuntime runtime, boolean loadInCache, long addre
     /**
      * Get a range of LogData from the server
      *
-     * This is using the underlying bulk read implementation for
+     * <p>This is using the underlying bulk read implementation for
      * fetching a range of addresses. This read will return
      * a map ordered by address.
      *
-     * It uses a ClosedOpen range : [start, end)
-     * (e.g [0, 5) == (0,1,2,3,4))
+     * <p>It uses a closed range: [start, end]
+     * (e.g [0, 5] == [0,1,2,3,4])
      *
      * @param start start address for the bulk read
-     * @param end end address for the bulk read
+     * @param end   end address for the bulk read
      * @return logData map ordered by addresses (increasing)
      */
     static Map<Long, ILogData> getLogData(CorfuRuntime runtime, long start, long end) {
```

Comment:
I don't think this is necessary. Can you inline it?

The `AddressSpaceView` should expose such API directly.

Reply:
Done.